src/Helpers.cpp

```diff
@@ -0,0 +1,66 @@
+#include "Helpers.hpp"
+
+namespace radar_base {
+    namespace helper_functions {
+
+        void iterateOverSweepsAndApplyFuntion(Radar const& radar_echo,
```

Comment:
Define namespace functions using the namespace explicitly (and correct spelling while you're at it, Funtion -> Function), e.g.

`helper_functions::iterateOverSweepsAndApplyFunction`

This guarantees that the declaration matches the definition. In this case, you could do

~~~
using namespace radar_base;

void helper_functions::....
~~~